### ISSUE TITLE
[Suggestion] Chose selector(s) for read on special site..

### ISSUE THREAD

**Issue by Smorvel** [ken107/read-aloud#130]
This feature allow you to choose selectors on sites, to show extension what it must read on current site by clicking on extension icon.

**Comment by ken107** [comment]
Are you talking about CSS selector?  Such as adding a CSS class to elements that should be read aloud?

**Comment by Smorvel** [comment]
@ken107 yes, for example for github i can add this selector `td.d-block.comment-body.markdown-body.js-comment-body>p` and when extension can find it by clicking on his icon, it start to read it. If selector not found, then it reading by default..

Also you can add regular expression, so i can set patterns of what not to read.. Like links etc.

**Comment by ken107** [comment]
so let's say this will work on GitHub, but it only applies for you?  Can someone else benefit from it?

**Comment by Smorvel** [comment]
@ken107 yes it applies only for user that did set it up for himself..

> Can someone else benefit from it?
From my settings? I think not.. but if you want, you can add functional to share own presets..

**Comment by ken107** [comment]
I think this's good.  Let me plan it out.  Thank you

**Comment by Smorvel** [comment]
@ken107 any progress?

**Comment by ken107** [comment]
I'll be sure to include it in the next release.  Thanks

**Comment by Smorvel** [comment]
@ken107 Can share some news about this update (about progress or plans)?
